### ISSUE TITLE
Errors if only client side tests are defined

### ISSUE THREAD

**Issue by mitar** [peerlibrary/meteor-classy-test#5]
Using 0.2.10 version and creating a test file one adds only on the client, an error is thrown in the test:

```
Server-side callable test failed.Error: Server callable 19 cannot be found.
```

**Comment by kostko** [comment]
Can you provide a sample test case that causes this error?

**Comment by kostko** [comment]
This has been fixed in 3dd3ede2493f4dfa0e8c5419bee4d1fcaaffbfda.
